content/docs/fairing/install-fairing.md

```diff
@@ -0,0 +1,239 @@
++++
+title = "Install Kubeflow Fairing"
+description = "Setting up your Kubeflow Fairing development environment"
+weight = 10
++++
+
+You can use Kubeflow Fairing to build, train, and deploy machine learning (ML)
+models in a hybrid cloud environment directly from Python code or a Jupyter
+notebook. This guide describes how to install Kubeflow Fairing in your
+development environment for [local development][local], or [development in a
+hosted notebook][hosted].
+
+## Using Kubeflow Fairing with Kubeflow notebooks
+
+Kubeflow notebook servers that are built from one of the standard Jupyter
+Docker images include Kubeflow Fairing and come preconfigured for using
+Kubeflow Fairing to run training jobs on your Kubeflow cluster.
+
+If you use a Kubeflow notebook server that was built from a custom Jupyter
+Docker image as your development environment, follow the instruction on
+[setting up Kubeflow Fairing in a hosted notebook environment][hosted].  
+
+## Set up Kubeflow Fairing for local development 
+
+Follow these instructions to set up Kubeflow Fairing for local development.
+This guide has been tested on Linux and Mac OS X. Currently, this guide has
+not been tested on Windows.
+
+### Set up Python
+
+1.  You need **Python 3.6** or later to use Kubeflow Fairing. To check if
+    you have Python 3.6 or later installed, run the following command:
+
+    ```bash
+    python3 -V
+    ```
+
+    The response should be something like this:
+
+    ```
+    Python 3.6.5
+    ```
+
+    If you do not have Python 3.6 or later, you can [download
+    Python](https://www.python.org/downloads/) from the Python Software
+    Foundation.
+
+1.  Use virtualenv to create a virtual environment to install Kubeflow
```

Comment:
Optional: Turn this into a codephrase because it is a command.

```suggestion
1.  Use `virtualenv` to create a virtual environment to install Kubeflow
```